app/controllers/membership_applications_controller.rb

```diff
@@ -1,5 +1,5 @@
 class MembershipApplicationsController < ApplicationController
-  before_action :get_membership_application, except: [:information, :index, :new, :create]
+  before_action :get_membership_application, except: [:information, :index, :new, :create, :export]
```

Comment:
No #export action in this controller (was probably here and then moved to AdminController?)